### PR TITLE
Feat/add pr r7

### PR DESCRIPTION
**Pull request by garloff**
Add press release to our web page, german and english.
Pushed to our staging branch, so we can review the rendered version on https://staging.scs.community/
Before merging, I'd like to merge the release notes for R7.
(If need be, we might add the link to the turnkey table into the release notes later the afternoon.)

### CODE REVIEW

**Comment by garloff** [comment]
For completeness: Here is the link to the release-notes PR:
https://github.com/SovereignCloudStack/release-notes/pull/27

File: _i18n/de/_posts/announcements/2024-09-11-release7.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+zur Verteilung von Kubernetes Control-Plane Knoten auf verschiedene Hosts im
+Cluster-API-Provider für OpenStack oder die durchgängige Verschlüsselung von
+internen Verbindungen in Kolla Ansible. Die beiden Beispiele werden zu
+Verbesserungen im nächsten Release führen und sind in R7 noch nicht enthalten.
+
+## Aktuelle Software
+
+SCS R7 kommt mit aktuellem OpenStack (2024.1 Caracal), welches auf verschiedenen
+Linux Distributionen betrieben werden kann. Weiterhin unterstützt wird Ubuntu
+22.04 LTS, jetzt neu: auch Ubuntu 24.04 LTS, Debian 12 und CentOS Stream 9. Die
```
Comment (fkr) [inline _i18n/de/_posts/announcements/2024-09-11-release7.md:57]: Question is wether we want to mention that as part of the CentOS support RHEL is also supported. RHEL support is important in various cases.

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+---
+# Turnkey Solution for Cloud Computing - Sovereign Cloud Stack Publishes Release 7
+
+**Berlin, 2024-09-11**: Turnkey solution for Cloud Computing – Sovereign Cloud Stack publishes release 7
+
+The Sovereign Cloud Stack (SCS) provides a complete set of base technology
+for the realization of digital sovereignty and open source strategy. SCS
+this way offers a digitally sovereign, secure, comprehensive, standardized,
+and open solution for the virtualization and container layers as base for
+containerized applications. The software implements the certifiable SCS
```
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:19]: I'd remove the germanization of the sentence.

```suggestion
for the realization of digital sovereignty and open source strategy. SCS
offers a digitally sovereign, secure, comprehensive, standardized,
and open solution for the virtualization and container layers as base for
containerized applications. The software implements the certifiable SCS
```

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+Kubernetes, SCS would be unthinkable. The members of the SCS community
+(both OSBA employees in the SCS project team as well as contractors and
+users) are active members of the respective upstream communities and
+contribute to the technology there. SCS' strategic direction is to
+avoid doing improvements exclusively in our own downstream implementation
+but puts focus on aligning with the upstream technology communities.
+This is sometimes slower, but results in a more sustainable development
+process. Current examples for this are contributions from the SCS
+community to the domain-manager role in OpenStack, the work on
+the Cluster-API-Provider ofr OpenStack to ensure that several control
```
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:48]:
```suggestion
process. Current examples are contributions from the SCS
community to the domain-manager role in OpenStack, the work on
the Cluster-API-Provider ofr OpenStack to ensure that several control
```

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+plane nodes of a Kubernetes cluster do not (accidentially) end up on the
+same underlaying hardware node (hypervisor) or the work on comprehensive
+encryption of internal connections in kolla-ansible. The first two
+examples will be part of the next release of SCS and have not yet trickled
+back to R7.
+
+## Current Software
+
+SCS R7 is shipped with current OpenStack (2024.1 Caracal) which can be operated
+on several Linux distrobutions: Ubuntu 22.04 LTS continues to be supported;
```
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:58]:
```suggestion
on several Linux distributions: Ubuntu 22.04 LTS continues to be supported;
```

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+Ubuntu 24.04 LTS, Debian 12 and CentOS Stream 9 habve been added. VPN-as-a-Service
+is now also supported and validated with the modern software-defined-networking
+technology OVN that we have used by default for some years now. The latest version
+of Ceph (Reef) and Deployemnt of Ceph via rook are available as Technical
```
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:62]:
```suggestion
of Ceph (Reef) and Deployment of Ceph via rook are available as Technical
```

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+Previews and can evaluated already as replacement for the proven Ceph Quincy
+deployed with ceph-ansible.
+
+A lot of improvments went into Cluster Stacks. They are using the current
```
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:66]:
```suggestion
A lot of improvements went into Cluster Stacks. They are using the current
```

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+version of Cluster API (v1.8) and the current Cluster-API-Provider for OpenStack
+(v0.10) which finally promises a stable API. Recent versions of Kubernetes
+(v1.30/1.31) have been validated. Cluster Stack can now also use any OCI registry
```
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:69]:
```suggestion
(v1.30/1.31) have been validated. Cluster Stacks can now also use any OCI registry
```

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+framework is as flexible as we designed it to be. On SCS infra, we have
+successfully enabled Kamaji (as a technical preview) as an option, where the
+control planes of several workload clusters can be in a shared cluster in order
+to reduce the overhead of creating many clusters. 
+
+The most important improvement under the hood of Cluster Stacks are the
+so-called multi-stage-addons. With their help, we can correctly deal
+with version dependencies between components, e.g. in case of cluster
+version upgrades. This helps to ensure a seamless availability of the
+cluster during upgrades. The old Kubernetes-as-a-Sevice v1 technology
```
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:86]:
```suggestion
cluster during upgrades. The old Kubernetes-as-a-Service v1 technology
```

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+
+## Operational Aspects and Security
+
+The OpenStack Health Monitor has proven really useful to monitor the
+virtualization layer in SCS. The code has aged though and a new implementation
+was created. It uses modern technology and can easily be extended with
+container scenario tests. Maintenance and further development has been
+simplified by this transition significantly.
+
+Another significant investment in securing the tehnology happened with
```
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:98]:
```suggestion
Another significant investment in securing the technology happened with
```

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+area of standards and certification.
+back in May, [it was shown](https://scs.community/2024/05/13/cost-of-making-an-openstack-cluster-scs-compliant/)
+that SCS-compatible IaaS-v4 compliance can be achieved with moderate effort
+using Yaook, an implementation that is completely independent of the SCS
+reference implementataion.
+New partnes have been added to the
+[list of SCS-Clouds](https://docs.scs.community/standards/certification/overview/#compliant-cloud-environments):
+AOV, SysEleven, and with Proof-of-Concept environments the KDO Service
+GmbH as well as Cloud&Heat Technologies GmbH. SysEleven is not using
+the SCS rewference implementation either. Next to the automated daily
```
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:116]:
```suggestion
the SCS reference implementation either. Next to the automated daily
```

File: _i18n/en/_posts/announcements/2024-09-11-release7.md (hunk 3 above)
Comment (fkr) [inline _i18n/en/_posts/announcements/2024-09-11-release7.md:48]:
```suggestion
the Cluster-API-Provider for OpenStack to ensure that several control
```

**Review by fkr — CHANGES_REQUESTED** [review]

File: _i18n/de/_posts/announcements/2024-09-11-release7.md (hunk 1 above)
Comment (garloff) [reply _i18n/de/_posts/announcements/2024-09-11-release7.md:57]: I have taken the information from the release notes.
We could write `CentOS Stream 9 (and thus also RHEL 9)`.

File: _i18n/en/_posts/announcements/2024-09-11-release7.md (hunk 3 above)
Comment (garloff) [reply _i18n/en/_posts/announcements/2024-09-11-release7.md]: Fixed in [811866e](https://github.com/SovereignCloudStack/website/pull/1018/commits/811866e4162e15c2a71d32202052ba9d5b2d9009)

File: _i18n/de/_posts/announcements/2024-09-11-release7.md [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+Cluster-API-Provider for OpenStack (v0.10) bereitgestellt und unterstützen
+aktuelle Kubernetes Versionen (v1.30/v1.31). Die Cluster Stacks ermöglichen jetzt die
+Nutzung von OCI-Registries, um eigene Versionen der Cluster Stacks zu nutzen.
+Eigene Node-Images können einfach erstellt und zur Nutzung auf einem
+Objektspeicher vorgehalten werden. Die Nutzung ist jetzt auch möglich, wenn die
+Infrastruktur mit Zertifikaten arbeitet, die nicht von einer der
+voreingestellten TLS Zertifizierungsstellen kommen.
+
+Neben der Nutzung von Cluster Stacks auf SCS-konformer Infrastruktur werden
+diese auch zunehmend auf der Hetzner Cloud genutzt. Dies stärkt die Technologie
```
[the remark this answers is not in the document]
Comment (garloff) [reply _i18n/de/_posts/announcements/2024-09-11-release7.md:74]: `diese zunehmend auch auf anderen Clouds (wie z.B. Hetzner) genutzt.`
If we are aware if  other examples, those could be mentioned as well, of course.

[the remark this answers is not in the document]
Comment (garloff) [reply _i18n/de/_posts/announcements/2024-09-11-release7.md]: OK, changed the text to just use Hetzner as example.

**Comment by garloff** [comment]
Reviews welcome, so we can push this out!

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+users) are active members of the respective upstream communities and
+contribute to the technology there. SCS' strategic direction is to
+avoid doing improvements exclusively in our own downstream implementation
+but puts focus on aligning with the upstream technology communities.
+This is sometimes slower, but results in a more sustainable development
+process. Current examples are contributions from the SCS
+community to the domain-manager role in OpenStack, the work on
+the Cluster-API-Provider for OpenStack to ensure that several control
+plane nodes of a Kubernetes cluster do not (accidentially) end up on the
+same underlaying hardware node (hypervisor) or the work on comprehensive
```
[the remark this answers is not in the document]
Comment (garloff) [reply _i18n/en/_posts/announcements/2024-09-11-release7.md]: Fixed with 647ff4e

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 13]
```diff
@@ -0,0 +1,149 @@
...
+The OpenStack Health Monitor has proven really useful to monitor the
+virtualization layer in SCS. The code has aged though and a new implementation
+was created. It uses modern technology and can easily be extended with
+container scenario tests. Maintenance and further development has been
+simplified by this transition significantly.
+
+Another significant investment in securing the technology happened with
+the support from specialized penetration testers. We also automated the
+penetration tests in a CI pipeline, so they are run regularly or to validate
+changes. This continous security assessment is an important contribution
```
[the remark this answers is not in the document]
Comment (garloff) [reply _i18n/en/_posts/announcements/2024-09-11-release7.md]: Fixed with 647ff4e

File: _i18n/en/_posts/announcements/2024-09-11-release7.md [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+to defend against growing cybersecurity threats.
+
+## SCS-compliant with Open Standards
+
+Next to the work on the reference implementation, there was progress in the
+area of standards and certification.
+back in May, [it was shown](https://scs.community/2024/05/13/cost-of-making-an-openstack-cluster-scs-compliant/)
+that SCS-compatible IaaS-v4 compliance can be achieved with moderate effort
+using Yaook, an implementation that is completely independent of the SCS
+reference implementataion.
```
[the remark this answers is not in the document]
Comment (garloff) [reply _i18n/en/_posts/announcements/2024-09-11-release7.md]: Fixed with 647ff4e

**Comment by garloff** [comment]
Thanks for the fixes, @gtema. Let's merge and publish ...

**Review by gtema — APPROVED** [review]

**Pull request merged**